server/py/services/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -74,27 +75,64 @@ def create_model_endpoint(
             * **archive**:
             1. If model endpoints with the same name exist, preserve them.
             2. Create a new model endpoint with the same name and set it to `latest`.
+        :param model_path:             The path to the model artifact.
 
-        :return: `ModelEndpoint` object.
+        :return: The crated `ModelEndpoint` object.
```

Comment:
```suggestion
        :return: The created `ModelEndpoint` object or `None` if the creation strategy is `SKIP`. 
```

Reply:
In addition add 
:raise: MLRunInvalidArgumentError if the creation strategy is not valid